### PR TITLE
Refresh description

### PR DESCRIPTION
**Pull request by mchf**
Bnc#1009931

### CODE REVIEW

File: src/modules/LanItems.rb [hunk 1]
```diff
@@ -1280,17 +1280,17 @@ def ip_overview(ip)
       when /DHCP/
         bullets << format("%s %s", _("IP address assigned using"), ip)
       else
-        prefixlen = NetworkInterfaces.Current["PREFIXLEN"]
-        if prefixlen
+        prefixlen = LanItems.prefix
```
Comment (jreidinger) [inline src/modules/LanItems.rb]: it looks a bit inconsistent. Is it prefix or length of prefix? variable is len, but method is only prefix

File: src/modules/LanItems.rb [hunk 2]
```diff
@@ -1366,26 +1366,32 @@ def startmode_overview(item_id)
       [startmode_descr]
     end
 
-    def ip_overview(ip)
+    # Creates details for device overvies based on ip configuration type
```
Comment (teclator) [inline src/modules/LanItems.rb]: overvie**w**

File: src/modules/LanItems.rb [hunk 3]
```diff
@@ -1366,26 +1366,32 @@ def startmode_overview(item_id)
...
+    #
+    # Produces list of strings. Strings are intended for "bullet" list, e.g.:
+    # * <string1>
+    # * <string2>
+    #
+    # return [Array] list of strings, one string is intended for one "bullet"
```
Comment (teclator) [inline src/modules/LanItems.rb]: please, document also the param

File: src/modules/LanItems.rb [hunk 4]
```diff
@@ -1418,15 +1424,16 @@ def BuildLanOverview
 
         if !ifcfg_name.empty?
           ifcfg_conf = GetDeviceMap(key)
+          log.error("BuildLanOverview: devmap for #{key}/#{ifcfg_name} is nil") if ifcfg.nil?
```
Comment (mchf) [inline src/modules/LanItems.rb]: yeah, details ;-) fixed

Comment (mchf) [inline src/modules/LanItems.rb]: when you know ```key```, you can get hwinfo from logs already.

**Comment by teclator** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for reviews

**Pull request merged**